go.mod

```diff
@@ -0,0 +1,69 @@
+module github.com/xenolf/lego
+
+require (
+	cloud.google.com/go v0.26.0 // indirect
+	github.com/Azure/azure-sdk-for-go v19.1.0+incompatible
+	github.com/Azure/go-autorest v10.15.2+incompatible
+	github.com/JamesClonk/vultr v0.0.0-20180101102256-fa1c0367800d
+	github.com/OpenDNS/vegadns2client v0.0.0-20180418235048-a3fa4a771d87
+	github.com/akamai/AkamaiOPEN-edgegrid-golang v0.6.2
+	github.com/aliyun/alibaba-cloud-sdk-go v0.0.0-20180828111155-cad214d7d71f
+	github.com/aws/aws-sdk-go v1.15.23
+	github.com/cloudflare/cloudflare-go v0.8.5
+	github.com/cpu/goacmedns v0.0.0-20180701200144-565ecf2a84df
+	github.com/decker502/dnspod-go v0.0.0-20180416134550-83a3ba562b04
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
+	github.com/dimchansky/utfbom v1.0.0 // indirect
+	github.com/dnaeon/go-vcr v0.0.0-20180920040454-5637cf3d8a31 // indirect
+	github.com/dnsimple/dnsimple-go v0.0.0-20180703121714-35bcc6b47c20
+	github.com/exoscale/egoscale v0.11.1
+	github.com/fatih/structs v1.1.0 // indirect
+	github.com/go-ini/ini v1.38.2 // indirect
+	github.com/google/go-querystring v0.0.0-20170111101155-53e6ce116135 // indirect
+	github.com/google/uuid v1.0.0 // indirect
+	github.com/gopherjs/gopherjs v0.0.0-20181103185306-d547d1d9531e // indirect
+	github.com/h2non/gock v1.0.11 // indirect
+	github.com/iij/doapi v0.0.0-20180911005243-8803795a9b7b
+	github.com/json-iterator/go v1.1.5 // indirect
+	github.com/jtolds/gls v4.2.1+incompatible // indirect
+	github.com/juju/ratelimit v1.0.1 // indirect
+	github.com/kolo/xmlrpc v0.0.0-20181023172212-16bdd962781d // indirect
+	github.com/kr/pretty v0.1.0 // indirect
+	github.com/ldez/go-auroradns v0.0.0-20181107021640-532168a1e313
```

Comment:
FYI https://github.com/ldez/go-auroradns/releases/tag/v2.0.1

Reply:
Aha, I'm afraid that the module name for `go-auroradns` is still invalid. I did not realize it is at `v2` already, so it needs `/v2` at the end of it's path.